### ISSUE TITLE
A problem when using format file for ruff in Windows and Linux 

### ISSUE THREAD

**Issue by DK05310214** [astral-sh/ruff-vscode#441]
I'm using Ruff for code formatting across different systems and have hit a snag with specifying a format file path in my settings. This setup causes an error when switching systems because the file path is not recognized on the new platform.

Is there a way to make Ruff dynamically adjust the file path based on the operating system it's running on? This would greatly enhance its usability in multi-OS environments.

![image](https://github.com/astral-sh/ruff-vscode/assets/62154505/e54fabfb-f1ce-4bcd-8590-45995bd13d7a)

**Comment by MichaReiser** [comment]
@DK05310214 no, Ruff has no such functionality. I understand that you want to configure different paths on different systems (/platforms?) because the file is stored at a different location. 

CC: @snowsignal I think this is interesting when thinking about the new settings

**Comment by DK05310214** [comment]
> @DK05310214 no, Ruff has no such functionality. I understand that you want to configure different paths on different systems (/platforms?) because the file is stored at a different location.
> 
> CC: @snowsignal I think this is interesting when thinking about the new settings

Thanks for reply. 
I think this feature may be very useful. 
The paths to configuration file can be set as a list. We can iterate through this list to search for configuration file and pass the file that found for the first time to ruff.

**Comment by MichaReiser** [comment]
Would it work if you place the file in your home directory and pass a relative path `--config ~/path/to/ruff.toml` ?
